tests/core/test_install.py

```diff
@@ -109,3 +119,35 @@ def test_install_multiple_packges(config, test_assets_dir, owned_pkg):
     assert check_dir_trees_equal(
         config.ethpm_dir, (test_assets_dir / "multiple" / "ethpm_packages")
     )
+
+
+@pytest.mark.parametrize("uninstall,keep", (("wallet", "owned"), ("owned", "wallet")))
+def test_uninstall_packages(
+    uninstall, keep, config, test_assets_dir, owned_pkg, wallet_pkg
+):
+    install_package(owned_pkg, config)
+    install_package(wallet_pkg, config)
+    uninstall_package(uninstall, config)
+
+    assert (config.ethpm_dir / keep).is_dir()
+    assert not (config.ethpm_dir / uninstall).is_dir()
+    assert check_dir_trees_equal(
+        config.ethpm_dir, (test_assets_dir / keep / "ipfs_uri" / "ethpm_packages")
+    )
+
+
+def test_uninstall_package_warns_if_package_doesnt_exist(config):
+    with pytest.raises(InstallError, match="Unable to uninstall"):
+        uninstall_package("invalid", config)
+
+
+def test_list(config, owned_pkg, wallet_pkg, caplog):
+    install_package(owned_pkg, config)
+    install_package(wallet_pkg, config)
+
+    with caplog.at_level(logging.INFO):
+        list_installed_packages(config)
+        assert "-- <Package wallet==1.0.0>" in caplog.text
```

Comment:
You might add the newline at the end of this text to be more precise in verifying these are properly formatted since the current format would allow them all munged on the same line.